inc/cachify.class.php

```diff
@@ -651,7 +651,7 @@ public static function add_dashboard_count( $items = array() )
 			esc_attr(strtolower($method)),
 			esc_html__( 'Caching method', 'cachify' ),
 			esc_attr($method),
-			( empty($size) ? esc_html__( 'Empty', 'cachify' ) : size_format($size) )
+			( empty($size) ? esc_html__( 'Empty Cache', 'cachify' ) : size_format($size) . ' ' . esc_html__( 'Cache', 'cachify' ) )
```

Comment:
Again a small hint, for the last request. Please use strict comparison coding rules `$size == 0` should be `$size === 0`.